assets/src/dashboard/app/views/templateDetails/index.js

```diff
@@ -210,10 +213,27 @@ function TemplateDetails() {
     createStoryFromTemplate(template);
   }, [createStoryFromTemplate, template]);
 
+  const handlePreviewTemplate = useCallback(
+    (previewTemplate) => {
+      previewVisible.set(previewTemplate);
+    },
+    [previewVisible]
+  );
+
   if (!template) {
     return null;
   }
 
+  if (previewVisible.value) {
+    return (
+      <PreviewStoryView
+        isTemplate
+        story={previewVisible.value}
+        handleClose={() => handlePreviewTemplate()}
```

Comment:
nit: This is canceling the use of `useCallback` and can just be `handleClose={handlePreviewTemplate}`

Reply:
This is a good call out because it shows that this is confusing. The reason I'd done this (and I'll update it in the other PR for explore templates as well) is because the first prop here on click is the event and we just want the template. In the useTemplateView util I'm using the existence of the template prop to set the  visibility of the template preview. But that does not come through with the above! It just looks like it's a redundancy!  

I've updated so that it can just be `handleClose={handlePreviewTemplate}` and am passing the event through as the first prop. I think this makes it clearer what's going on.